package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What's this?  I know about hs_discover/package-lock.json, but not sure what this one is for.

Reply:
This is standard frontend dev, you can look it up and I will also reach out to you on this.